js/layout-javascript/news-feed-code.js

```diff
@@ -105,7 +105,15 @@ DynamicList.prototype.toggleFilterElement = function (target, toggle) {
 
 DynamicList.prototype.attachObservers = function() {
   var _this = this;
+
   // Attach your event listeners here
+  Fliplet.Hooks.on('beforePageView', function (options) {
+    if (options.addToHistory === false) {
+      _this.closeDetails();
```

Comment:
any chance `closeDetails` can have the `Fliplet.Page.Context.remove('dynamicListOpenId');` in it?

Reply:
Updated